Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabIntegrationManager.cpp

```diff
@@ -720,6 +720,12 @@ namespace AzToolsFramework
                         m_prefabOverridePublicInterface->AreOverridesPresent(selectedEntity))
                     {
                         QAction* revertAction = menu->addAction(QObject::tr("Revert Overrides"));
+                        revertAction->setToolTip(QObject::tr("Revert all overrides on this entity to whatever is in the prefab file."));
+                        if (m_prefabOverridePublicInterface->GetOverrideType(selectedEntity) == OverrideType::AddEntity)
+                        {
+                            revertAction->setToolTip(QObject::tr("Cannot revert overrides on entities that are added as overrides."));
```

Comment:
Suggestion: indicate that user can delete the entity instead.

Reply:
Sure, I was considering adding (Use delete instead) at the end of the tooltip, but it felt a little long. I'll add it though

Reply:
Is there a way to just make it show up and instead just _be_ a delete?  Least confusing?  Or still bad?

Reply:
The concern I have is that most users are unlikely to understand the fact that the "Revert" will actually just make that entity go away. Asking them to delete instead makes the operation clear